### PR TITLE
Use TypeScript Declaration Merging to get rid of some `any`s

### PR DESCRIPTION
**Pull request by nperez0111**
This adds some properties that were being accessed on the window object, onto the window object

I'm not completely certain that I got all of the types 100% correct, but I pulled them from the two places that I see them being set:
 - https://github.com/continuedev/continue/blob/1e75a7e8ff9b438c36c9d783d8c849ec12815d94/extensions/vscode/src/debugPanel.ts#L722-L733
 - https://github.com/continuedev/continue/blob/1e75a7e8ff9b438c36c9d783d8c849ec12815d94/extensions/intellij/src/main/kotlin/com/github/continuedev/continueintellijextension/toolWindow/ContinuePluginToolWindowFactory.kt#L124-L128

### CODE REVIEW

File: extensions/intellij/src/main/kotlin/com/github/continuedev/continueintellijextension/toolWindow/ContinuePluginToolWindowFactory.kt [hunk 1]
```diff
@@ -123,7 +123,7 @@ class ContinuePluginToolWindowFactory : ToolWindowFactory, DumbAware {
                             val jsonData = mutableMapOf(
                                     "type" to "onLoad",
                                     "windowId" to continuePluginService.windowId,
-                                    "workspacePaths" to continuePluginService.worksapcePaths,
+                                    "workspacePaths" to continuePluginService.workspacePaths,
```
Comment (nperez0111) [inline extensions/intellij/src/main/kotlin/com/github/continuedev/continueintellijextension/toolWindow/ContinuePluginToolWindowFactory.kt:126]: Just caught a typo here and figured it was worth addressing

**Comment by sestinj** [comment]
This is fabulous, and the fact that a typo was caught too...this is exactly why we need strong typing : )

I think it's ready for merge if you can confirm?

**Comment by nperez0111** [comment]
Yes, this is ready for merging @sestinj!

**Comment by sestinj** [comment]
awesome, good work!

**Pull request merged**